e2e/runTests.js

```diff
@@ -0,0 +1,104 @@
+import child_process from 'child_process'; // eslint-disable-line
+import fs from 'fs';
+import https from 'https';
+import os from 'os';
+import path from 'path';
+import {promisify} from 'util';
+
+import glob from 'glob';
+import Mocha from 'mocha';
+
+const execP = promisify(child_process.exec);
+const globP = promisify(glob);
+
+const SERVER_TIME_TO_START = 10000;
```

Comment:
I'd suggest that any of these "big knob" constants are kept in their own file.

Reply:
Yessir! I'd love to be able to fork the server process and receive a message back when the server is ready to accept connections. If you've got any ideas, let me know! I'll be looking into this in my current refactor.

Reply:
Yessir! I'd love to be able to fork the server process and receive a message back when the server is ready to accept connections. If you've got any ideas, let me know! I'll be looking into this in my current refactor.